### ISSUE TITLE
not able to find report-template.hbs

### ISSUE THREAD

**Issue by shortcut2yp** [ortoniKC/ortoni-report#1]
Added ortoni-report 1.0.7 as dev dependency and added it in reporter config. 
On running the test cases getting below error. 

Error in reporter Error: ENOENT: no such file or directory, open '/Users/automation-playwright/report-template.hbs'

Seems this is added in .gitignore file of the repo.

**Comment by ortoniKC** [comment]
Hi, hbs file should be within the dist folder, which is not included in the git.
If you go to node_modules/ortoni-report/dist you should see this. 
Could you please share the playwright.config reporting part to understand the issue.

**Comment by shortcut2yp** [comment]
```
reporter: [['list', { printSteps: true }],["html",{open:"never"}],['json', {  outputFile: 'results.json' }],['ortoni-report']]
```

**Comment by ortoniKC** [comment]
Yes, I am also facing the same issue. Thanks for the report. 
I will update now.

**Comment by ortoniKC** [comment]
Released version 1.0.8, it should be fixed. pls check.

**Comment by shortcut2yp** [comment]
working now ! Thanks 🙌 
Somehow I feel it is creating a new folder each time at top level , seems issue with naming convention of the screenshot folder. Do check if its same for you. Can close this issue. 

<img width="491" alt="Screenshot 2024-06-16 at 11 00 08 AM" src="https://github.com/ortoniKC/ortoni-report/assets/32234622/801f5c25-ce0c-4464-8e94-0003bc2b3718">

**Comment by ortoniKC** [comment]
In my test it is like 
![image](https://github.com/ortoniKC/ortoni-report/assets/58769833/c38f90fb-275d-4b77-81c9-29a85915801d)
Pls confirm if you're able to see the screenshots within the report?

I will try to debug the path issue if exists, 
Kindly share the report in your LinkedIn if possible. 
Thanks.

**Comment by shortcut2yp** [comment]
<img width="1512" alt="Screenshot 2024-06-16 at 11 21 15 AM" src="https://github.com/ortoniKC/ortoni-report/assets/32234622/f6d8b45e-6f0d-4bd3-b717-39114848de91">


report looks like this.  Seems some issue with my config , will check. 

Sure once I generate proper report , will share in linkedln 🙌

**Comment by shortcut2yp** [comment]
can it be because I am using UNIX based system and the code has 

``` const screenshotsDir = path.join(process.cwd(), 'screenshots\\' + test.id);})```

whereas UNIX uses ```/```  to determine folder structure. 

can we use ```const screenshotsDir = path.join(process.cwd(), 'screenshots', test.id);``` in place of above code ? 


```
The backslash (\\) is treated as a literal character in the string. This means the path separator is explicitly Windows-style, regardless of the operating system. If run on a Unix-like system (Linux, macOS), the backslash might not be interpreted as a path separator, leading to potential issues.
```

**Comment by ortoniKC** [comment]
Great, I am providing a fix.

**Comment by ortoniKC** [comment]
Fixed with version 1.0.9
Also custom attributes like projectName, authorName & testType - https://www.npmjs.com/package/ortoni-report?activeTab=readme
Do check it out & if any issue pls create a new one. 
Thanks for the support.
